src/libecalc/dto/result/tabular_time_series.py

```diff
@@ -110,6 +111,12 @@ def resample(self, freq: Frequency) -> Self:
                 else:
                     # NOTE: Operational settings are not resampled. Should add support?
                     pass
+
+            # Resampling compressor stream results:
+            elif isinstance(values, dto.result.results.CompressorStreamConditionResult):
```

Comment:
Can't you just say `if isinstance(values, TimeSeries) or isinstance(values, TabularTimeSeries):` on line 101 and drop this?

Reply:
Yes of course, nice catch:-)